docs/modules/tile-converter/cli-reference/tile-converter.md

```diff
@@ -85,6 +85,7 @@ NodeJS 14 or higher is required.
 | generate-bounding-volumes | \*                        |                           | Create new OBB and MBS bounding volumes from geometry instead of conversion it from the source tile bounding volume                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
 | metadata-class            | \*                        |                           | Set metadata class related to EXT_feature_metadata or EXT_structural_meatadata. See [Schema class](#schema-class-selection) extensions                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| analyze-only              | \*                        |                           | Analyze the input tileset content without conversion                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
Maybe a `--analyze` / `--convert` type API would be nicer than keep adding `--analyze-only` special cases.

Since we are nearing a major version release you could potentially get away with making some changes (though it is always best to maximize backwards compatibility).

Reply:
Done